tests/keras/backend/backend_test.py

```diff
@@ -39,9 +39,12 @@
     KMX = None
     warnings.warn('Could not import the MXNet backend')
 
+BACKENDS = set(BACKENDS)
+BACKENDS_WITHOUT_MXNET = BACKENDS - set([KMX])
```

Comment:
So these tests does not include MXNet backend?

Reply:
That is correct. Tests using BACKENDS_WITHOUT_MXNET will not run for MXNet.